.vscode/settings.json

```diff
@@ -14,27 +14,15 @@
   "html.format.wrapLineLength": 120,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.fixAll.stylelint": true
+    "source.fixAll.eslint": "explicit",
+    "source.fixAll.stylelint": "explicit"
```

Comment:
What is the difference between `true` and `"explicit"`? Why is `"explicit"` needed in this case?

Reply:
i actually didn't make this change (it happened automatically), but `"explicit"` seems to be the correct value.

this is the only documentation i could find: https://github.com/microsoft/vscode-eslint#version-204